### ISSUE TITLE
[Open Call] Contribute to the Course of AI Kit with Raspberry Pi: From Zero to Hero

### ISSUE THREAD

**Issue by MatthewJeffson** [Seeed-Studio/wiki-documents#1765]
# Play your AI Kit from Beginner to Expert

This course is designed to teach you how to harness the power of AI on the Raspberry Pi, with a particular focus on using an AI kit to perform essential computer vision tasks. Throughout the course, you'll learn how to integrate AI into real-world IoT (Internet of Things) applications, from object detection and image classification to more complex visual recognition tasks. 

We will guide you step-by-step through setting up your Raspberry Pi, using AI frameworks, and deploying these models in various practical scenarios. 

Whether you are a hobbyist, a student, or a professional, this course will provide you with the foundational knowledge and hands-on experience necessary to bring AI-driven solutions to life on resource-constrained devices like the Raspberry Pi.

## 📢 Open Call for Contributors

We here invite developers and experts to contribute to our Tutorial-of-AI-Kit-with-Raspberry-Pi project on GitHub!

## How Do We Work with Contributors?
We welcome contributions from anyone interested in collaboration via GitHub. This open call is part of our [Contributor Project](https://github.com/orgs/Seeed-Studio/projects/6/views/1?pane=issue&itemId=83150125). If this is your first time joining us, click and read to learn how the project works.

- Assignments: We offer a variety of assignments to enhance wiki content, each with a detailed description.
- Submission: Contributors can submit their content via a Pull Request after completing the assignments.
- Review: Maintainers will merge the submission and record the contributions.
- Collaboration is key to our success, and we look forward to working with our contributors to make this platform the best it can be.

## What Do We Offer?
We express our gratitude to contributors by offering a range of rewards based on the difficulty of the assignment (Tier 0/1/2/3), the expected completion time (1/3/7/15 days), the actual submission time, and the quality of the content. This task is listed as Tier 2. 

**Rewards: Contributors receive a $200 cash bonus as a token of appreciation.**
Acknowledgment: All contributors and their contributions will be acknowledged and showcased on this platform.
We believe our contributors are an integral part of our success, and we aim to recognize and reward their efforts appropriately.

### How to Contribute

1. **Visit the GitHub Repository:**
   - Go to our [Course GitHub repository](https://github.com/Seeed-Projects/Tutorial-of-AI-Kit-with-Raspberry-Pi-From-Zero-to-Hero).
 
2. **Choose Contents or Topic:**
   - Select some contents that interest you, such as computer vision, robotics, generative AI, or any other area in the guide.

3. **Fork and Clone the Repository:**
   - Fork the repository to your GitHub account, then clone it to your local machine. Before starting working on your contribution, we highly recommend you submit the outline first to let us know your plan. We will give merge suggestions! 

4. **Develop and Test:**
   - Write, improve, or review content, and ensure it's tested and compatible with the Raspberry Pi platform.

5. **Submit a Pull Request:**
   - Once your contribution is ready, submit a pull request. Make sure to include a clear description of your changes.

### Ready to Get Started?

Dive into the project, share your skills, and contribute to building the future of AI and robotics education. We’re excited to see what you create!

For any questions or further information, feel free to reach out via the GitHub issues page or contact edge@seeed.cc

**Comment by feiticeir0** [comment]
This seems very nice ! Kudos to SeeedStudio for doing this. 
I would love to help, if I had a AI kit. Have to get one !

**Comment by MatthewJeffson** [comment]
@feiticeir0 That would be great Bruno! I will arrange the product deliver:D

How is everything going with you latelyXD

Best Regards,
Matthew

**Comment by chilu18** [comment]
Hello my github name is @chilu18 - other github accounts are @heysalad @agrisavedev :)

**Comment by chilu18** [comment]
@chilu18

**Comment by MatthewJeffson** [comment]
Thank you so much! Peter! :D @chilu18

**Comment by Mechmallow** [comment]
The last post in chapter 1 is about LLM, can I add an introduction to embodied intelligence?

**Comment by MatthewJeffson** [comment]
Yes! Definitely welcome! @Mechmallow Thank you for the contribution! 
Best Regards,
Matthew
